### PR TITLE
Export build date, tag and hash properties for release

### PR DESCRIPTION
**Pull request by adrcunha**
* use UTC time for dates for consistency
* remove double initialization of parameters

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -108,10 +108,13 @@ This is a helper script for Knative E2E test scripts. To use it:
    if the default values don't fit your needs:
 
    - `E2E_CLUSTER_REGION`: Cluster region, defaults to `us-central1`.
-   - `E2E_CLUSTER_BACKUP_REGIONS`: Space-separated list of regions to retry test cluster creation in case of stockout. Defaults to `us-west1 us-east1`.
+   - `E2E_CLUSTER_BACKUP_REGIONS`: Space-separated list of regions to retry test
+     cluster creation in case of stockout. Defaults to `us-west1 us-east1`.
    - `E2E_CLUSTER_ZONE`: Cluster zone (e.g., `a`), defaults to none (i.e. use a regional
      cluster).
-   - `E2E_CLUSTER_BACKUP_ZONES`: Space-separated list of zones to retry test cluster creation in case of stockout. If defined, `E2E_CLUSTER_BACKUP_REGIONS` will be ignored thus it defaults to none.
+   - `E2E_CLUSTER_BACKUP_ZONES`: Space-separated list of zones to retry test cluster
```
Comment (adrcunha) [inline scripts/README.md]:
```suggestion
   - `E2E_CLUSTER_BACKUP_ZONES`: Space-separated list of zones to retry test cluster
```

File: scripts/README.md [hunk 2]
```diff
@@ -241,14 +244,20 @@ This is a helper script for Knative release scripts. To use it:
    - `RELEASE_GCS_BUCKET`: contains the GCS bucket name to store the manifests if
      `--release-gcs` was passed, otherwise the default value `knative-nightly/<repo>`
      will be used. It is empty if `--publish` was not passed.
+   - `BUILD_COMMIT_HASH`: the commit short hash for the current repo. If the current
+     git tree is dirty, it will have `-dirty` appended to it.
```
Comment (ericKlawitter) [inline scripts/README.md]: should this be `--dirty`?

Comment (adrcunha) [reply scripts/README.md]: No. It's a suffix of a string, not a flag.

**Pull request merged**